.github/workflows/versioning.yml

```
@@ -25,21 +27,36 @@ jobs:
+          git checkout -b "$TEMP_BRANCH"
+
       - name: Bump Version
         run: |
           chmod +x ./versioning/bump_version.sh
           ./versioning/bump_version.sh
 
-      - name: Push Changes and Tags
+      - name: Push Changes to Temporary Branch
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:33: Double quote to prevent globbing and word splitting

(shellcheck)